Traceback (most recent call last):
  File "/opt/code/federated_learning.py", line 109, in <module>
    run()
  File "/opt/code/federated_learning.py", line 97, in run
    with open(CODE_PATH+'federated_learning.json') as data_file:    
FileNotFoundError: [Errno 2] No such file or directory: 'federated_learning.json'
